3-probability/01-chance-intro/notes.qmd

```diff
@@ -387,39 +390,31 @@ We can see that the (blue) *sample proportions* look similar to the (gold) *popu
+What we have seen here is how when we draw at random, we get a sample that resembles the population, that is, a *representative sample*. This is called an *empirical* histogram, that is, a histogram derived from data.
 
 
-**The Box Model**
+#### Tossing a biased coin
 
-The purpose of creating a box model is to understand the variability in a *chance process* such as tossing a coin over and over and counting the number of heads, or sampling voters to ask them if they will vote for President Biden in the 2024 elections. We do this by having the tickets represent the possible outcomes associated with the chance process, and using the variability in the tickets that are drawn to understand the variability of the chance process.
+Suppose I have a coin that is twice as likely to land heads as it is to land tails. This means that I cannot represent $\Omega$ as $\{H, T\}$ since heads and tails are not equally likely. How should I write $\Omega$ so that the outcomes are equally likely?
 
-In order to create a box model that will model a process in which we have a repeated action (such as tossing a coin or drawing a ticket from a box), we must specify:
+In this case, we want to represent equally likely outcomes, and want $H$ to be twice as likely as $T$. We can therefore represent $\Omega$ as $\{H, H, T \}$. Now the chance of the coin landing $H$ can be written as $P(A)$ where $A$ is the event the coin lands $H$ is given by 2/3. 
 
-1. The box and tickets (what tickets go in the box, and the numbers written on them)
-2. The number of draws
-3. Whether or not we replace tickets between draws
-4. How we will summarize the values on the drawn tickets (we will sum the draws or compute their average)
+Suppose we toss the coin twice. How would we list the outcomes so that they are equally likely?
+Now we have to be careful, and think about all the things that can happen on the second toss if we have $H$ on the first toss.
 
-Note that these tickets should be identical in every way, except the value written on them, and  assume that the tickets in the box are *equally likely* to be drawn (we draw without looking into the box).
+This is much easier to imagine if we imagine drawing *twice* from a box of tickets, but putting the first ticket back before drawing the second (to represent the fact that the probabilities of landing $H$ or $T$ stay the same on the second toss.)
 
-### Setting up a box model: examples
-Let's practice creating boxes to model coin tosses and die rolls. For each of the cases below, specify the four items above.
+Now, imagine the box of tickets that represents $\Omega$ to be $\fbox{H, H, T}$. 
+We draw one ticket at first, which could be one of three tickets (there are two tickets that could be $H$, and one $T$). We can represent it using the following picture:
 
-##### Example 1: Tossing a coin
 
-**Tossing a fair coin once**
+![](images/biased-coin.png){fig-align=center width="600"}
```

Comment:
Here is where it is erroring out. You have not committed this image, so it can't find it.